NewForce-at-Mountwest/bangazon-workforce-management-the-kiwi-lagoons
# Bg employee create

### Description

I updated the Employee Controller.
Created a view model for Employee to get a list of departments for a drop down.
Verbs supported are Create.
Users are able to create a new employee.  They will be able to enter the first name and last name, mark a checkbox if the employee is a supervisor, and select a department from a dropdown list.

## Related Ticket(s)

https://trello.com/c/WzFrX5XG/2-2-hr-should-be-able-to-add-an-employee

## Steps to Test

- git fetch --all  and checkout the bg-employee-create branch.

- Run the following SQL query in the BangazonWorkforce database:  https://github.com/NewForce-at-Mountwest/bangazon-workforce-management-the-kiwi-lagoons/blob/bg-employee-create/BangazonWorkforceMVC/BangazonWorkforceMVC/SQLQuery3.sql

- Click on the employees link in the navbar.

- Click "Create New Employee" 

- Fill out the first name and last name.

- Check the Supervisor checkbox.

- Select a department from the drop down and click create.

- You should see the new employee in the employees list.

- Go to VS Community and do a SQL query of  SELECT * FROM Employee

- If you checked the Supervisor checkbox, make sure the IsSupervisor bool is true for the employee you created.  It should be false if you didn't mark the checkbox.

## Checklist:

- [x] I checked out bg-employee-create.
- [x] After creating a new employee I saw the employee show up in the employees list.
- [x] In VS Community I did a SQL query for SELECT * FROM Employee
- [x] The IsSupervisor bool is correct.  (True if you checked the box, false if you did not)
